### DISCUSSION TITLE
support termux

### DISCUSSION THREAD

**Discussion by Jzhenli** [beeware/briefcase#2000] [Q&A]
Hi, if it's possible to run briefcase on [termux](https://github.com/termux/termux-app)?  if yes, maybe we can build apk more easily for andriod.

**Comment by freakboy3742** [comment]
I'm not sure I understand how supporting termux would make building APKs more simple - assuming it is even possible to do so, building an APK on the device doesn't address most of the issues of distributing an APK to other users.

**Reply by Jzhenli** [reply]
On the termux platform, for the Non-pure Python packages which are written in C, C++, Rust, or another language(such as Pydantic)  can be compiled to android  machine code easily.

**Reply by freakboy3742** [reply]
Sure - but that doesn't make anything *Briefcase* does easier. That's an issue about building Android wheels. Briefcase *uses* Android wheels, but doesn't compile them.
